exporter/azure_monitor/example/otherClientExample.go

```diff
@@ -0,0 +1,32 @@
+package main
+// Package: Runs code for using Azure exporter
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+
+	"go.opencensus.io/plugin/ochttp"
+	"go.opencensus.io/exporter/azure_monitor/creator"
+)
+
+func main() {
+	ctx := context.Background() // In other usages, the context would have been passed down after starting some traces.
+	creator.EnableObservabilityAndExporter()
+	req, _ := http.NewRequest("GET", "https://en.wikipedia.org/wiki/Chicken", nil)
+	// It is imperative that req.WithContext is used to
+	// propagate context and use it in the request.
+	req = req.WithContext(ctx)
+	client := &http.Client{Transport: &ochttp.Transport{}}
+	res, err := client.Do(req)

```

Comment:
Where is the span started in this example? How does this example work differently from the other client one?

Reply:
The span here is produced when the response body closes in line 30 I believe. If I delete that line, there is no span. I found this example online. It gives a span of type client which the server.go and client.go files did not give me. They give a span of type server. The main.go example gives a normal span that is neither type (https://godoc.org/go.opencensus.io/trace#pkg-constants). I couldn't find how the span kind type is defined.